Trigger replication failing (#2195)
neo4j-contrib/neo4j-apoc-procedures: docs/asciidoc/modules/ROOT/partials/triggers.adoc

[source,properties]
----
apoc.trigger.enabled=true
apoc.trigger.refresh=60000
----

.Description
[opts=header,cols="3"]
|===
| Option Key | Value | Description
| apoc.trigger.enabled | true/false, default false | Enable/Disable the feature

> | apoc.trigger.refresh | number, default 60000 | Value in ms for trigger replication check across cluster nodes

Reviewer: ```suggestion
| apoc.trigger.refresh | number, default 60000 | Interval in ms after which a replication check is triggered across all cluster nodes
```

Author: done
